docs/Changelog.md

```diff
@@ -20384,13 +20384,12 @@ This version of the operator has been available since version 16 of the default
 
 ### <a name="Where-16"></a>**Where-16**</a>
 
-  Return elements, either from X or Y, depending on condition
-      (with Numpy-style broadcasting support).
-      Where behaves like numpy.where with three parameters:
-      https://docs.scipy.org/doc/numpy/reference/generated/numpy.where.html
+  Return elements, either from X or Y, depending on condition.
+      Where behaves like
+      [numpy.where](https://docs.scipy.org/doc/numpy/reference/generated/numpy.where.html)
+      with three parameters.
   
-  **History**
-  - Version 16 adds bfloat16 to the types allowed (for the second and third parameter).
```

Comment:
it is bit of challenge to easily figure out what changed in a particular op version... while this is not an ideal solution the history section makes it easier for folks to understand what changed

Reply:
Thanks!